contracts/farms/common/StakingTokenHolder.sol

```diff
@@ -267,9 +261,8 @@ abstract contract RevoFarmBot is ERC20, AccessControl, Pausable {
             "Sending fees failed"
         );
 
-        // reinvest LPs and adjust FP weight
+        // Adjust FP weight and send compounder bonus
         uint256 lpEarnings = lpBalance - compounderFee - reserveFee;
-        _deposit(lpEarnings);
         lpTotalBalance += lpEarnings;
```

Comment:
whoops, I forgot this was here when I suggested we rename it to issueFees. I could see this leading to bugs if we forget it's a side effect of issuing fees (which would be pretty reasonable). I think we should either take it out, or rename the method to make it more clear this is happening

Reply:
I would probably vote for renaming to be more clear I think; I like the idea of encapsulating all interactions with the `RevoFees` contract into a single method in the base class, rather than splitting it into multiple. That way, a farm can call one method and take care of everything related to fees/payouts.